OasisDEX/oasis-borrow
# feat: remove team members

# [Remove Team Members](https://app.shortcut.com/oazo-apps/story/12081/update-https-summer-fi-about-remove-leavers)
  
## Changes 👷‍♀️
 - Removed Sam and Adam from `About->Team` page
  
## How to test 🧪
 - Run the FE and go to `About->Team` page. Confirm that neither Adam nor Sam are there
